app/ecosystem/fixture.ts

```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
+      'A developer integration toolkit to let users buy/sell crypto in any app, website or web plugin.',
+    categories: ['dev-tools', 'infra'],
+    twitter: 'https://x.com/transak',
+    website: 'https://transak.com/'
+  },
+  {
+    image: '/img/ecosystem/zeeve.png',
+    title: 'Zeeve',
+    description:
+      'An enterprise-grade automation platform providing deployment, monitoring and management of Blockchain nodes and networks.',
+    categories: ['infra', 'ecosystem-partner'],
+    twitter: 'https://x.com/0xZeeve',
+    website: 'https://www.zeeve.io/'
+  },
+  {
+    image: '/img/ecosystem/Web3Auth.jpg',
+    title: 'Web3Auth',
+    description:
+      'Pluggable wallet infrastructure for Web3 wallets and applications. Streamline the onboarding of both mainstream and crypto native users in under a minute. ',
+    categories: ['dev-tools'],
+    twitter: 'https://x.com/web3auth',
+    website: 'https://web3auth.io/'
+  },
+  {
+    image: '/img/ecosystem/Secret-Network.jpg',
+    title: 'Secret Network',
+    description:
+      'decentralized confidential computing (DeCC), enabling dApps to make use of encrypted data.',
+    categories: ['dev-tools'],
+    twitter: 'https://x.com/SecretNetwork',
+    website: 'https://scrt.network/'
+  },
+  {
+    image: '/img/ecosystem/Layer3.jpg',
+    title: 'Layer3',
+    description:
+      'Discover the world of crypto like never before. With interactive Quests that make learning and exploring crypto fun, engaging, and rewarding.',
+    categories: ['social'],
+    twitter: 'https://x.com/layer3xyz',
+    website: 'https://layer3.xyz/'
+  },
+  {
+    image: '/img/ecosystem/MintyCodeDAO.jpg',
+    title: 'Mintycode.io',
+    description:
+      'Decentralised Marketplace for Open-Source Creators, Users, and Contributors.',
+    categories: ['social'],
+    twitter: 'https://x.com/MintyCodeDao',
+    website: 'http://mintycode.io/'
+  },
+  {
+    image: '/img/ecosystem/UHT.jpg',
+    title: 'UHT',
+    description:
+      'Gamify health, reward healthy behaviour & lifestyle habits with UHT (Universal Health Token).',
+    categories: ['social'],
+    twitter: 'https://x.com/UHTofficial',
+    website: 'https://universalhealthtoken.xyz/'
+  },
+  {
+    image: '/img/ecosystem/Open Ticketing Protocol Logo.jpg',
+    title: 'Open Ticketing Protocol (GET Protocol)',
+    description:
+      'The home of onchain ticketing. Bringing accessible onchain benefits to millions of event attendees, organizers and artists around the world.',
+    categories: ['social'],
+    twitter: 'https://x.com/onopenxyz',
+    website: 'https://onopen.xyz/'
+  },
+  {
+    image: '/img/ecosystem/tachyswap.jpg',
+    title: 'TachySwap',
+    description: 'AMM based decentralized exchange.',
+    categories: ['defi'],
+    twitter: 'https://x.com/tachyswap',
+    website: 'https://tachyswap.org/'
+  },
+  {
+    image: '/img/ecosystem/cobweb-labs.jpg',
+    title: 'CobWeb Labs',
+    description: 've(3,3) DEX.',
```

Comment:
Can you do any better with the description here?

Reply:
Let me double check with Aaron